about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <script defer src="script.js"></script>
+    <title>About</title>
+</head>
+<body id="about-body">
+    <nav class="page-nav">
+        <ul class="nav-contacts">
+            <li class="nav-contats-li">
+                <a href="#">
+                    <img src="icons/facebook.png" alt="facebook" class="nav-contacts-icon">
+                </a>
+            </li>
+            <li class="nav-contats-li">
+                <a href="#">
+                    <img src="icons/twitter.png" alt="facebook" class="nav-contacts-icon">
+                </a>
+            </li>
+            <li class="nav-contats-li">
+                <a href="#" class="nav-contacts-text">
+                    English
+                </a>
+            </li>
+            <li class="nav-contats-li">
+                <a href="#" class="nav-contacts-text">
+                    My Page
+                </a>
+            </li>
+            <li class="nav-contats-li">
+                <a href="#" class="nav-contacts-text">
+                    Logout
+                </a>
+            </li>
+        </ul>
+        <div class="nav-links">
+            <div class="menu-btn">
+                <div class="menu-btn__burger"></div>
+            </div>
+            <div class="nav-items">
+                <a href="index.html" class="logo-link">
+                    <img src="icons/page-logo.png" alt="ITech-logo" class="header-logo" >
+                </a>
+                <ul class="nav-ul">
+                    <li class="nav-li">
+                        <a href="#" class="nav-link">About</a>
+                    </li>
+                    <li class="nav-li">
+                        <a href="#" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-li">
+                        <a href="#" class="nav-link">Join</a>
+                    </li>
+                    <li class="nav-li">
+                        <a href="#" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-li">
+                        <button href="#" class="nav-btn">ITech campain</button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <section class="section-1 about-section1">
+        <h2 class="page-greeting">"Hello sharing world"</h2>
```

Comment:
- [ ] Kindly note that the footer of the `about me` section should have a dark background color like in the original design.

![image](https://user-images.githubusercontent.com/98453979/219663035-1ef857d5-62de-4685-bc89-bee30b9c9d50.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219663424-6d009035-dac4-49bf-b5c1-ac1de7949b4a.png)